__test__/getCommand.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { it, describe, expect } from 'vitest'
+import getCommand from '../utils/getCommand'
+
+describe("should generate correct command", () => {
```

Comment:
I would use `getCommand` as the`describe` name, and then `should generate the correct command for yarn` as the `it` name